wfl/calculators/orca.py

```diff
@@ -172,8 +173,11 @@ def evaluate_op(atoms, base_rundir=None, dir_prefix="ORCA_",
         try:
             at.calc.calculate(at)
             calculation_succeeded = True
-        except CalculationFailed:
-            pass
+            if 'DFT_FAILED_ORCA' in at.info:
```

Comment:
Are these be coming from previous failed attempts?

Reply:
Yes, just in case it was set in an earlier attempt.